projects/playground/e2e/integration/playground.spec.ts

```diff
@@ -0,0 +1,158 @@
+    cy.get('[data-cy=login-popup]').should('not.be.checked');
+  });
+
+  it('shows default logged in options', () => {
+    cy.visit('/');
+    cy.get('#login').should('be.visible').click();
+
+    loginToAuth0();
+
+    cy.get('[data-cy=logout-local]').should('not.be.checked');
+    cy.get('[data-cy=logout-federated]').should('not.be.checked');
+    cy.get('[data-cy=accessToken-ignoreCache]').should('not.be.checked');
+    cy.get('[data-cy=accessToken-silently]').should('be.checked');
+    cy.get('[data-cy=accessToken-popup]').should('not.be.checked');
+
+    cy.get('#logout').should('be.visible').click();
+  });
+
+  it('do redirect login and show user and access token', () => {
+    cy.visit('/');
+    cy.get('#login').should('be.visible').click();
+
+    cy.url().should('include', 'https://brucke.auth0.com/login');
+    loginToAuth0();
+
+    cy.get('[data-cy=userProfile]').contains(`"email": "${EMAIL}"`);
+
+    cy.get('[data-cy=accessToken]').should('be.empty');
+    cy.get('#accessToken').click();
+    cy.get('[data-cy=accessToken]')
+      .should('not.be.empty')
+      .invoke('text')
+      .then((token) => {
+        cy.get('#accessToken').click();
+        cy.get('[data-cy=accessToken]').should('have.text', token);
+      });
+
+    cy.get('#logout').should('be.visible').click();
+    cy.get('#login').should('be.visible');
+  });
+
+  it('do redirect login and get new access token', () => {
+    cy.visit('/');
+    cy.get('#login').should('be.visible').click();
+
+    cy.url().should('include', 'https://brucke.auth0.com/login');
+    loginToAuth0();
+
+    cy.get('[data-cy=userProfile]').contains(`"email": "${EMAIL}"`);
+
+    cy.get('[data-cy=accessToken]').should('be.empty');
+    cy.get('[data-cy=accessToken-ignoreCache]').check();
+    cy.get('#accessToken').click();
+    cy.get('[data-cy=accessToken]')
+      .should('not.be.empty')
+      .invoke('text')
+      .then((token) => {
+        cy.get('#accessToken').click();
+        cy.get('[data-cy=accessToken]')
+          .should('not.be.empty')
+          .and('not.have.text', token);
+      });
+
+    cy.get('#logout').should('be.visible').click();
+    cy.get('#login').should('be.visible');
+  });
+
+  it('do local logout (auth0 session remains valid)', () => {
+    cy.visit('/');
+    cy.get('#login').should('be.visible').click();
+
+    cy.url().should('include', 'https://brucke.auth0.com/login');
+    loginToAuth0();
+
+    cy.get('[data-cy=logout-localOnly]').check();
+    cy.get('#logout').click();
+
+    cy.get('#login').should('be.visible').click();
+    cy.url().should('include', 'https://brucke.auth0.com/login');
+    // logs in with the "last time you logged in with" button
+    cy.get('.auth0-lock-last-login-pane > a').should('be.visible').click();
+
+    cy.get('#logout').should('be.visible').click();
+    cy.get('#login').should('be.visible');
+  });
+
+  it('do federated logout (auth0 session is cleared)', () => {
```

Comment:
Do we want to keep this test given that we can't verify that we _actually_ do a federated logout?

If it's just to test that the options are specified, we have unit tests that cover that.

Reply:
I think there's value keeping it. Maybe we can rename the test to no longer make reference to the "federated" mode. We can even skip checking that checkbox and leaving both boxes unchecked. That would do a normal non-local logout. WDYT?

Checkboxes aside, what this test is asserting is that the auth0 session is cleared. How it does that is by checking that after logging out, visiting the login page will not remember who logged in previously.

This is how the "second login" looks for this test:
![image](https://user-images.githubusercontent.com/3900123/92402554-215c0d00-f106-11ea-9bbc-577417ba8e87.png)

And this is how it looks for the "local logout" test:
![image](https://user-images.githubusercontent.com/3900123/92402627-3df84500-f106-11ea-8fcf-c1775d4a265a.png)

Reply:
Yeah I think that's better suited to a normal non-local logout, I'd be happy with that change 👍

Reply:
Done. I'm not "unchecking" any checkboxes because they are not checked by default. That default state is also tested in one of our first tests.